app/javascript/shared/components/tags.jsx

```diff
@@ -22,6 +22,7 @@ const NAVIGATION_KEYS = [
 ];
 
 const LETTERS_NUMBERS = /[a-z0-9]/i;
+export const DEFAULT_TAG_FORMAT = '[0-9A-Za-z, ]+';
```

Comment:
Previously, this lived in `article-form/components/TagsField.jsx` but it's used only by the listings tag search. For now I've moved it here so it's co-located with other tag code used by listings, but I think a follow up PR to this one will be to add the same autocomplete to the listings form so that we can remove all this other tags code.